api/revapi.json

```diff
@@ -27,7 +27,13 @@
     "criticality" : "highlight",
     "minSeverity" : "POTENTIALLY_BREAKING",
     "minCriticality" : "documented",
-    "differences" : [ ]
+    "differences" : [
+        {
+            "regex": true,
+            "code": ".*",
```

Comment:
@jponge see how to tame revapi!

Reply:
Actually, I can clear them because we've already released a 4.2.0 version